collect/collector.go

```diff
@@ -0,0 +1,24 @@
+// Package collect implements a library to create
+// and define different pull based collectors.
```

Comment:
are we using the collectors at all?

Reply:
No, I just put these here as place holders for future work. Should I delete them?